app/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>meow</title>
+</head>
+<body ng-app="meowApp">
+  <meowsay></meowsay>
+  <meowsay></meowsay>
+  <meowsay></meowsay>
+  <meowsay></meowsay>
+  <meowsay></meowsay>
```

Comment:
Not losing points, but lots of ``meowsays`` goign on in here. For UX design, best to just keep the one template in that contains your two first cat and the second cat that you used for bonus.